misc/config_tools/config_app/requirements

```diff
@@ -2,3 +2,4 @@ Flask==1.1.1
 flask_bootstrap==3.3.7.1
 xmlschema==1.4.1
 lxml==4.6.2
+defusedxml==0.7.1
```

Comment:
Ditto, can we remove `lxml` from the file?

Reply:
What this shows too is we have this list of python dependencies in too many places.  Best is to put them in one ``requirements.txt`` file and use that everywhere.

Reply:
@yuchuyang We can do this "single list in one place" fix in another PR if you need to merge this one for continuing your security work.